core/Enclave/enclave_hsm.cpp

```diff
@@ -225,19 +361,19 @@ sgx_status_t enclave_create_aes_key(uint8_t *cmk_blob, uint32_t cmk_blob_size, u
     if (cmk_blob == NULL || cmk_blob_size != real_blob_len)
         return SGX_ERROR_INVALID_PARAMETER;
 
-    uint8_t* tmp = (uint8_t *)malloc(SGX_AES_KEY_SIZE);
+    uint8_t* tmp = (uint8_t *)malloc(key_size);
```

Comment:
sgx_get_key_size() may return FFFFFFFF to key_size.

Reply:
Yes, it should check return code.